frontend/src/components/Tools/Tools.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {
+    Card, CardImg, CardText, CardBody,
+    CardTitle, CardSubtitle, Button
+  } from 'reactstrap';
+
+export default function Tools(){
+    return(
+        <div>
+            <p className="text-center">Tools used in this project: </p>
+            <div className="container">
+                <div className="row">
+                    <div className="col">
```

Comment:
What do you think about using reactstrap [Container,Row,and Column](https://reactstrap.github.io/components/layout/) components instead of giving the classNames?

Reply:
Sounds good, I haven't considered that. I will make those changes on my next commit.